react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt

```diff
@@ -48,8 +49,8 @@ internal class PaywallFooterViewManager : BasePaywallViewManager<PaywallFooterVi
                 val finalWidth = maxWidth.coerceAtLeast(suggestedMinimumWidth)
                 val finalHeight = maxHeight.coerceAtLeast(suggestedMinimumHeight)
                 setMeasuredDimension(finalWidth, finalHeight)
-                (context as? ThemedReactContext)?.let { themedReactContext ->
-                    themedReactContext.runOnNativeModulesQueueThread {
+                (context.applicationContext as? ReactContext)?.let { context ->
```

Comment:
Here the correct fix would be:

```
(context as? ThemedReactContext)?.reactApplicationContext?.let { themedReactContext ->
```

Reply:
Right, that works! Thank you

The new architecture is still not working though... Do you know how to fix this error I am seeing in the logcat?

```                                                                                               com.facebook.react.bridge.ReactNoCrashBridgeNotAllowedSoftException: getNativeModule(UIManagerModule.class) cannot be called when the bridge is disabled
```

Reply:
> Do you know how to fix this error I am seeing in the logcat?

Which error? The log you sent is empty

Reply:
sorry, there was a weird tab character that was breaking the rendering. Just fixed it

```
com.facebook.react.bridge.ReactNoCrashBridgeNotAllowedSoftException: getNativeModule(UIManagerModule.class) cannot be called when the bridge is disabled
```

Reply:
You can't invoke `getNativeModule(UIManagerModule.class)` on Bridgeless as you're trying to access the old rendered (which won't be there anymore).

I believe this is caused by this `onMeasure` method you added here:
https://github.com/RevenueCat/react-native-purchases/blob/3ec67636b97aa963131333530a4c83a0fa7c994e/react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallFooterViewManager.kt#L39-L57

I suggest you remove this method entirely (or guard it to run only on Old Architecture). Custom measuring in New Architecture is done differently (it's done in C++) and it's something you generally don't need to do.

Reply:
If I don't override the `onMeasure` the React Native view is not displayed. `finalWidth` and `finalHeight` have the correct height in that code, but the react native view's size is 0, that's why we call `updateNodeSize`.

This was the only solution we could find to correctly display the view. We got inspired by https://medium.com/traveloka-engineering/react-native-at-traveloka-native-ui-components-c6b66f789f35

Do you maybe have another suggestion of why the view's size is 0? We don't want devs to have to define a size in Javascript for this view, since the size is dynamic and depends on the content. To give more context, this is how the  view looks like (it's that view anchored at the bottom on top of the text):

![image](https://github.com/RevenueCat/react-native-purchases/assets/664544/22465e7c-6231-4084-8c63-7ce3701bdff3)

Reply:
I understand your situation. 
My suggestion would be to follow the same approach used by React Native Slider here:
- https://github.com/callstack/react-native-slider/pull/589

Please let me know if you get stuck and I can try to help you @vegaro

Reply:
Ended up with another approach https://github.com/RevenueCat/react-native-purchases/pull/980 which works in all architectures with and without bridge